pennylane/operation.py

```diff
@@ -735,11 +823,6 @@ def __matmul__(self, other):
 
         raise ValueError("Can only perform tensor products between observables.")
 
-    @property
-    def eigvals(self):
-        r"""Returns the eigenvalues of the observable"""
-        raise NotImplementedError
```

Comment:
Just double checking, but this is now removed because it's part of the base Operator class?